README.rst

```diff
@@ -75,8 +75,46 @@ that the tests pass and also the code follows the code style properly. The
 Configuring and Running
 =======================
 
-To be done... integrade is just starting and in the future you can see how to
-configure and run the functional tests in this section.
+Integrade can be configured to test any instance of cloudigrade. The
+**REQUIRED** environment variables are::
+
+    CLOUDIGRADE_BASE_URL # base url without http/https prefix
+    CLOUDIGRADE_TOKEN    # This is the token of the super user created for
+                         # cloudigrade
+
+The **OPTIONAL** environment variables are::
+
+    CLOUDIGRADE_API_VERSION # defaults to 'v1'
+    CLOUDIGRADE_USER        # defaults to 'admin'
+    CLOUDIGRADE_USER_PASS   # at the moment we only use the token to
+                            # authenticate, but this will be useful later
+    USE_HTTPS  # defaults to false so communication is done over http.
+               #  Set to True to use https.
```

Comment:
SSL_VERIFY below uses false all lower case and this use True with capitalized word. I think we could consistent and maybe use both lowercase since True and False are Python boolean objs.

Reply:
Yeah, the comment was more in the sense of being consistent and said about using lower case in the examples because it is easier to type